packages/mobile/src/components/AccountOverview.tsx

```diff
@@ -70,8 +70,11 @@ export class AccountOverview extends React.Component<Props> {
     return fontSize
   }
 
-  refreshBalances = () => {
-    this.props.refreshAllBalances()
+  componentWillUpdate() {
```

Comment:
I don't think triggering based on this lifecycle method is the best way to go. Keep in mind that the component will only update if it requires a re-render. 
My feeling is that we should just cut this feature entirely. It's a bandaid to fix the issue of us not reliably fetching balance after all relevant transactions. WDYT?

Reply:
💯 agree that the balance update should be triggered on relevant events. However, I think this feature is a nice fallback in case we somehow miss one of the events and makes the app independent of the implementation of the stability fees.

And the way it's currently implemented, if one of the triggers makes the app refresh the balances then the balance wont refresh automatically until the balance is old enough.